spring-xsuaa-test/src/test/java/com/sap/cloud/security/xsuaa/test/JwtGeneratorTest.java

```diff
@@ -77,8 +75,8 @@ public void testTokenWithAttributes() {
 				.addAttribute(ANOTHER_ATTRIBUTE, new String[] { ANOTHER_ATTRIBUTE_VALUE, ANOTHER_ATTRIBUTE_VALUE_2 })
 				.getToken();
 		Map<String, Object> attributes = jwt.getClaimAsMap("xs.user.attributes");
-		assertThat((JSONArray) attributes.get(DUMMY_ATTRIBUTE), contains(DUMMY_ATTRIBUTE));
-		assertThat((JSONArray) attributes.get(ANOTHER_ATTRIBUTE),
+		assertThat(new JSONArray((ArrayList) attributes.get(DUMMY_ATTRIBUTE)), contains(DUMMY_ATTRIBUTE));
```

Comment:
In case one would still use minidev lib, then it would still work, right?

Reply:
I would expect it to work, yes.